### ISSUE TITLE
Updating ETT theme to 3.2.0, side effects in CSS

### ISSUE THREAD

**Issue by PhilReedData** [FAIRdom/fairdom.github.io#113]
I'd like to update the [version of ETT theme](https://github.com/ELIXIR-Belgium/elixir-toolkit-theme/releases/tag/3.2.0) from 2.3.0 to 3.2.0, to take advantage of bug fixes (resizing columns before page load completes, and more). It makes sense to do this before making any layout changes; we're looking at incorporating a page logo in the front-matter, so that it appears at the side of the h1 heading, not below, on pages such as BioIndustry 4.0 where we show the logo of the project near the top. I'll raise that separately.

When I tested changing the version number (locally) in `_config.yml`, there were some side effects, including:

1. Headings are larger (h1, h2 and h3); propose we restore them to current sizes.
2. 'Sections' boxes heading text have changed colour, so the heading text is not visible (too similar to background); propose we restore to current colours.
3. 'Sections' boxes contents have changed colour (fg and bg) when hovering, which is a bit too vivid; propose we either restore or make this effect much less vivid (eg keep dark text on light background).

I can fix these in CSS but it seemed significant enough that I should flag and see if anyone else wants to test for any further side effects. Note, I've not committed or written any changes yet.

![image](https://github.com/user-attachments/assets/ffa0a25c-75d7-401d-8025-80a6dbb414ef)

**Comment by bedroesb** [comment]
Hello Phil! Make sure to check the theme variables you set in _custom_variables.css. There has been some renamings if you check the release notes. To find the complete list please look here: https://github.com/ELIXIR-Belgium/elixir-toolkit-theme/blob/main/_sass/_variables.scss (So for many of theme no custom css classes needed!)

**Comment by PhilReedData** [comment]
Thanks @bedroesb, I've been able to update the custom variables and I'm happy with the styles now (working in a new branch, [ett-3.2.0](ett-3.2.0)). However, I still cannot get the heading sizes right. For example, h1 on any page, using Chrome Inspector:

- With 2.3.0: `h1, .h1 { font-size: calc(1.405rem + 1.86vw); }` Computes to 43.0702px
- With 3.2.0: `h1, .h1 { font-size: calc(1.455rem + 2.46ww); }` Computes to 49.3314px
- We have the bootstrap line: `$h1-font-size: 1rem * 2.8 !default;` which does nothing in 3.2.0. I change it to any other value, I use it in custom variables script with `!important` and it does nothing. 
- Similar with h2 and h3. I don't know where the font-size calculation is coming from.

Could you please help, what I am doing wrong with setting h1, h2, h3 sizes? The [ETT docs say I can use bootstrap variables for heading size](https://elixir-belgium.github.io/elixir-toolkit-theme/custom_branding#theme-styling), but I have [also read](https://stackoverflow.com/questions/10643107/what-does-default-in-a-css-property-value-mean) that it's better to use a custom variables file, though neither is working for me. Many thanks.

![image](https://github.com/user-attachments/assets/5f547ee1-38ba-4cb9-8042-1b2841ac3b91)

**Comment by bedroesb** [comment]
@PhilReedData Deffinatly use the Bootstrap variables for this! Since we make use of the [dynamic heading resizing](https://getbootstrap.com/docs/5.3/getting-started/rfs/) bootstrap offers. Example at the RDMkit: https://github.com/elixir-europe/rdmkit/blob/master/_sass/_bootstrap_variables.scss#L28>

Golden rule is, if you find a variable in bootstrap, it is always better to change that one (the source) than to overwrite it using custom rules.

**Comment by bedroesb** [comment]
Still centered h1 I see :D

**Comment by PhilReedData** [comment]
Good to know, thanks. After some more work, I learned that when using ETT 3.0 or greater, the `!default` part of the value stopped the `$h1-font-size` variable from working anywhere. (We had `$h1-font-size: 1rem * 2.8 !default;` etc).

When I took that out, it worked, but I had to adjust the multiplying factors a little higher to match their previous sizes.

The left-aligned h1 is coming, I'm going to do that next in a separate thread, while I'm implementing the site logos (did you have any more thoughts on my last email?).
